docs/Antora/modules/howto_guides/pages/howto_configure_streaming.adoc

```diff
@@ -0,0 +1,298 @@
+= Configure Streaming
+
+Typically, in addition to publishing its own structure (xref:glossary:glossary.adoc#signal[Signal]s,
+xref:glossary:glossary.adoc#channel[Channel]s, xref:glossary:glossary.adoc#function_block[Function Block]s, etc.),
+an openDAQ(TM) xref:glossary:glossary.adoc#device[Device] also publishes information about
+xref:glossary:glossary.adoc#streaming[Streaming] protocols it supports. This information includes the
+streaming protocol ID (e.g. 'daq.wss' or 'daq.ns'), which doubles as a prefix in the connection string,
+enabling the routing of streaming instantiation to the appropriate openDAQ(TM)
+xref:glossary:glossary.adoc#module[Module]. Additionally, it includes the device's IP address (used if the
+streaming protocol requires a network connection) and a range of optional parameters (e.g. port number).
+When a streaming server starting on the device, this automatically generates and prepares mentioned
+information to be published along with the device's structural details.
+
+As a result, when connecting an openDAQ(TM) device, a streaming connection can be established automatically
+using the published streaming connection details.
+
+[#streaming_config_per_device]
+== Configure Streaming per Device
+
+The xref:glossary:glossary.adoc#device[Device] configuration xref:glossary:glossary.adoc#property_object[Property
```

Comment:
In general, I'd avoid linking to the glossary, but link to the corresponding background info articles instead.

Reply:
Glossary links in Streaming how-to and in Streaming background are replaced with links to background info articles